README.md

```diff
@@ -5,10 +5,7 @@
 
 `@ibm-watson/assistant-web-chat-react` is a React library to extend the [web chat](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-deploy-web-chat) feature of [IBM Watson Assistant](https://www.ibm.com/cloud/watson-assistant) within your React application. This makes it easier to provide user-defined response types written in React, add content to custom elements with React, have the web chat and your site communicate more easily, and more.
 
-There are two utility functions provided by this library:
-
-1. The `WebChatContainer` function is a functional component that makes use of `withWebChat` to load web chat when the component is mounted.
-2. The `withWebChat` function creates a high-order-component (HOC) that you can wrap around an existing component to inject `createWebChatInstance` into it so your component can create a new instance of web chat when appropriate. You can find more information in the [withWebChat documentation](./WITH_WEB_CHAT.md).
```

Comment:
We should probably leave some mention of the old withWebChat with a note that it is going away and not recommended for use in case someone currently using it comes back looking for the docs.